concepticondata/conceptlists.tsv

```diff
@@ -373,16 +373,17 @@ Carvalho-2021-206	de Carvalho, Fernando O.	2021		206	basic, areal	English	Iñapa
 Koelle-1854-280	Koelle, Sigismund W.	1854		280	basic	English	African languages		Koelle1854		This large concept list, partitioned into 280 items which are subdivided on several occasions, was used to describe and compare the lexicon of 300 African languages.		PolyglottaAfricana
 Scherer-2013-20	Scherer, Klaus R. and Shuman, Vera and Fontaine, Johnny and Soriano, Cristina	2013		20	specific	English	Global		Scherer2013		The study introduces the Geneva emotion wheel which  is a measurement instrument for the self-report of emotional experience. It differs from instruments that require respondents to rate their feelings on a list of emotion words in that it organizes the emotion words to be rated in a theoretically justified two-dimensional system formed by the valence and power dimensions.		Geneva Emotion Wheel
 Jonauskaite-2020-12	Jonauskaite, Domicele and Abu-Akel, Ahmad and Dael, Nele and Oberfeld, Daniel and Abdel-Khalek, Ahmed M. and Al-Rasheed, Abdulrahman S. and Antonietti, Jean-Philippe and Bogushevskaya, Victoria and Chamseddine, Amer and Chkonia, Eka and Corona, Violeta and Fonseca-Pedrero, Eduardo and Griber, Yulia A. and Grimshaw, Gina and Hasan, Aya Ahmed and Havelka, Jelena and Hirnstein, Marco and Karlsson, Bodil S. A. and Laurent, Eric and Lindeman, Marjaana and Marquardt, Lynn and Mefoh, Philip and Papadatou-Pastou, Marietta and Pérez-Albéniz, Alicia and Pouyan, Niloufar and Roinishvili, Maya and Romanyuk, Lyudmyla and Salgado Montejo, Alejandro and Schrag, Yann and Sultanova, Aygun and Uusküla, Mari and Vainio, Suvi and Wąsowicz, Grażyna and Zdravković, Sunčica and Zhang, Meng and Mohr, Christine	2020		12	specific, colors	English,Arabic,Azerbaijani,Chinese,Dutch,Estonian,Finnish,French,German,Georgian,Greek,Hebrew,Italian,Lithuanian,Norwegian,Persian,Polish,Russian,Serbian,Spanish,Swedish,Ukrainian	Global		Jonauskaite2020		The study investigates color-emotion associations by using the Geneva emotion wheel [(Scherer et al. 2013)](:bib:Scherer2013) in combination with 12 color terms across 22 languages.		
-Goddard-2002-59	Goddard, Cliff and Wierzbicka, Anna	2002		59	specific	English	Global		Goddard2002		This list contains concept labels for 59 semantic primes sorted into different categoried based on [Natural Semantic Metalanguage](https://de.wikipedia.org/wiki/Natural_Semantic_Metalanguage)) approach. The authors claim the concepts to be universal.	14	
+Goddard-2002-59	Goddard, Cliff and Wierzbicka, Anna	2002		59	specific	English	Global		Goddard2002		This list contains concept labels for 59 semantic primes sorted into different categorized based on [Natural Semantic Metalanguage](https://de.wikipedia.org/wiki/Natural_Semantic_Metalanguage)) approach. The authors claim the concepts to be universal.	14	
```

Comment:
Thanks for noticing this typo! But I intended to write "categorie**s**" instead of "categorized" :)

Reply:
Whops, sorry! Will fix momentarily.